cli/npm/dsl.d.ts

```diff
@@ -10,6 +10,7 @@ type PrecRightRule = { type: 'PREC_RIGHT'; content: Rule; value: number };
 type PrecRule = { type: 'PREC'; content: Rule; value: number };
 type Repeat1Rule = { type: 'REPEAT1'; content: Rule };
 type RepeatRule = { type: 'REPEAT'; content: Rule };
+type ReservedRule = { type: 'RESERVED'; content: Rule; context_name: string };
```

Comment:
looks like this should also be added to the `Rule` type union on line 19 below. I'm getting the following TS error:

> Argument of type 'ReservedRule' is not assignable to parameter of type 'RuleOrLiteral'.
>  Type 'ReservedRule' is not assignable to type 'RegExp | SymbolRule<string> | AliasRule | BlankRule | ChoiceRule | FieldRule | ImmediateTokenRule | ... 10 more ... | RustRegex'.